### PR TITLE
Fix accessibility fundamentals page issues

### PR DESCRIPTION
**Pull request by jenny-s51**
Towards #3302 

Fixed md related issues. Remaining issues are related to the content in the image - if we want to make those changes to the image, it looks like we may need to open a design issue

### CODE REVIEW

File: packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md [hunk 1]
```diff
@@ -14,57 +14,64 @@ When it comes to product experiences, no one should get left behind. That's why
...
-- Alternative text (alt text) makes images accessible to users with low bandwidth connections or with older technologies that can’t load the images. 
-- Closed captioning benefits those in crowded areas, those teaching children how to read, or those learning a new language. 
+- Accessible written content is easier to comprehend for people who aren’t fluent in the language and for people with low literacy levels.
+- Alternative text (alt text) makes images accessible to users with low bandwidth connections or with older technologies that can’t load the images.
+- Closed captioning benefits those in crowded areas, those teaching children how to read, or those learning a new language.
 
 There are plenty more examples we could include. Essentially, accessibility takes _all users_ into account.
 
-<img src="./PF-a11y.png" alt="Accessibility spectrum example" />
+The image below shows that accessibility is a spectrum. Different kinds of cognitive and physical impairments impact how users interact with products. It is important to take all these differing disabilities into account when building accessible products.
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md:66]: A `pf-v5-u-screen-reader` class on this paragraph might make more sense. I don't think it needs to be a visible paragraph. WDYT, @jessiehuff ?

File: packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md [hunk 2]
```diff
@@ -14,57 +14,64 @@ When it comes to product experiences, no one should get left behind. That's why
...
+
+Users who have difficulty processing information benefit from well-written content.
+
+- Information should be clear, concise, and easy to scan.
 - Consider visual hierarchy, chunk content into short, related sections, and avoid long paragraphs.
 <br />
 <br />
 
-These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios. 
+These are some of the main user groups, but it is not an exhaustive list. Often “the Persona Spectrum”, coined by Microsoft, can be used to understand related mismatches and motivations across a spectrum of permanent, temporary, and situational scenarios.
```
Comment (nicolethoen) [inline packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md:56]: I think, according to the original issue, this line should be changed from `Often "the Persona Spectrum"` to `Often the "Persona Spectrum"`

File: packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md (hunk 1 above)
Comment (jenny-s51) [reply packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md]: Great idea @nicolethoen - added `pf-v5-u-screen-reader`

Comment (jessiehuff) [reply packages/v4/patternfly-docs/content/developer-resources/accessibility-guide.md]: I'm ok with either approach! When I added that paragraph, I think I was just trying to give context to that image in general, but I'm ok with making it only visible to screen readers if it doesn't make sense to include for visual users/is already obvious.

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**